packages/unlock-js/src/PublicLock/v10/getKeyExpirationByLockForOwner.js

```diff
@@ -12,15 +12,23 @@ export default async function (lockAddress, owner, network) {
     this.providerForNetwork(network)
   )
 
-  // We need to get the id.
-  const tokenId = await getTokenIdForOwner.bind(this)(
-    lockAddress,
-    owner,
-    network
+  const numberOfKeys = lockContract.balanceOf(owner)
```

Comment:
Actually I think `balanceOf` only returns the number of _valid_ keys. Not sure if that's an issue though? cc @clemsos

Reply:
Yes only valid keys (with a proper fix to be released w lock v11)